Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -459,6 +481,89 @@ def create_results(domain_result):
     return outputs
 
 
+def create_domain_risk_results(domain_result: dict):
+    """
+    Creates all the context data necessary given a domain result
+    Args:
+        domain_result (dict): DomainTools domain data
+
+    Returns: dict {
+        domain: <Common.Domain> - Domain indicator object with Iris results that map to Domain context
+        domaintools: <dict> - DomainTools context with all Iris results
+    }
+
+    """
+    domain = f"{domain_result.get('domain')}"
+
+    domain_risk_score_details = {
+        "overall_risk_score": domain_result.get("risk_score"),
```

Comment:
Consider using `None` instead of an empty string (`""`) as the default value for optional numeric scores.